test/show.jl

```diff
@@ -269,3 +269,12 @@ end
 @test_repr "bitstype 100 B"
 @test repr(:(bitstype A B)) == ":(bitstype A B)"
 @test repr(:(bitstype 100 B)) == ":(bitstype 100 B)"
+
+oldout = STDOUT
```

Comment:
I understand about the let block, but shouldn't it have the `@test` to make sure that it returns as expected?